src/app/routes.tsx

```diff
@@ -35,35 +37,12 @@ const Support = (routeProps: RouteComponentProps) => {
   );
 };
 
-const RouteWithTitleUpdates = ({ component: Component, isAsync = false, title, ...rest }) => {
-  const lastNavigation = useLastLocation();
-
-  function routeWithTitle(routeProps: RouteComponentProps) {
-    return (
-      <DocumentTitle title={title}>
-        <Component {...rest} {...routeProps} />
-      </DocumentTitle>
-    );
-  }
-
-  React.useEffect(() => {
-    if (!isAsync && lastNavigation !== null) {
-      routeFocusTimer = accessibleRouteChangeHandler();
-    }
-    return () => {
-      clearTimeout(routeFocusTimer);
-    };
-  }, []);
-
-  return <Route render={routeWithTitle} />;
-};
-
 export interface IAppRoute {
-  label: string;
+  label?: string;
   component: React.ComponentType<RouteComponentProps<any>> | React.ComponentType<any>;
-  icon: any;
+  icon?: any;
   exact?: boolean;
-  path: string;
+  path?: string;
```

Comment:
Why is `path` optional? It doesn't look like it's required for the title changes and a route with an optional path seems odd?

Reply:
Good question, I did this because the [404 route](https://github.com/patternfly/patternfly-react-seed/blob/master/src/app/routes.tsx#L105) I think shouldn't have a path

https://reacttraining.com/react-router/web/example/no-match

This also required me to use this null checking suppressor thingie, which I kinda felt weird about. Is there a better way?

https://github.com/patternfly/patternfly-react-seed/pull/64/files#diff-18dedd8c973dc9fe9edda131a31cf0d9R53